publish.sh

```diff
@@ -72,7 +69,7 @@ function build_clusters() {
 
       cluster-provision/gocli/build/cli provision --phases k8s cluster-provision/k8s/$i --slim
       ${CRI_BIN} tag ${TARGET_REPO}/k8s-$i ${TARGET_REPO}/k8s-$i:${KUBEVIRTCI_TAG}-slim
-    elif [ $ARCH == "s390x" ] && [ $i == "1.30" ]; then
+    elif [[ "$ARCH" == "s390x" && ( "$i" == "1.30" || "$i" == "1.31" ) ]]; then
```

Comment:
I guess these conditions can be removed when we include the k8s-1.32 provider and remove k8s-1.29 ?

Reply:
right, I think whereever we use version of provider in publish.sh, I think we can remove them